src/isomorphic/classic/types/__tests__/ReactPropTypes-test.js

```diff
@@ -40,7 +43,9 @@ function typeCheckPass(declaration, value) {
     props,
     'testProp',
     'testComponent',
-    ReactPropTypeLocations.prop
+    ReactPropTypeLocations.prop,
+    null,
+    ReactPropTypesSecret
```

Comment:
I learned that the hard way once everything started failing after moving this from production to dev 😄